src/utils/test-identification.util.ts

```diff
@@ -19,10 +19,10 @@ export class TestIdentification
     }
 
     public static captureEnvironment(): void {
-        ReportUtility.addLabel("OS", "TODO");
+        ReportUtility.addLabel("OS", (browser.capabilities as any).platformName);
```

Comment:
"browser" object (in lowercase) belongs to WDIO library and would be good to do not spread it's usage across the library and limit the calls to be only on files under the "wdio" folder. This should faciliate (if needed) changing to another e2e framework in the future.

Thus, I would suggest to use the "Browser" class (under "wdio" folder) to wrap the access to the browser capabilities. It would be great to add to this class methods such as:
- getOperatingSystem(): string
- getName(): string // For browser name
- getVersion(): string // For browser version

Reply:
sure! I'm trying to use the status() function in order to retrieve accurate info about the operating system (version, not only the name).

Reply:
Awesome, re-reviewed new changes and look great.